styles/globals.css

```diff
@@ -46,7 +52,7 @@ a {
 footer {
   width: 100%;
   text-align: center;
-  position: absolute;
+  position: fixed;
```

Comment:
![Screenshot (473)](https://user-images.githubusercontent.com/70262783/109605096-a91d5400-7b4a-11eb-9925-0c257ab95158.png)

@SBagaria2710  Save for later content gets hide behind the footer.